### ISSUE TITLE
Multiple package.jsons in project

### ISSUE THREAD

**Issue by whitecolor** [stealjs/steal#665]
Is steal able to handle multiple package.json schenarios when apps parts (call it modules) has their own package.json with their own deps:

```
/base
  package.json
  /module1
    node_modules
    package.json
  /module2
    node_modules
    package.json
```
?

**Comment by matthewp** [comment]
No, we have the expectation of there only being one package.json/node_modules folder. I'm not sure how easy it would be to add support for this either, i'm sure it's possible but we have several lookup objects that key on package names, so we'd have to change those so that they split along a specific package.json.  It's a good feature to have though, but I think someone other than me would have to take on the work to get it implemented.
